source/_components/script.markdown

```diff
@@ -29,11 +31,49 @@ Script names (e.g., `message_temperature` in the example above) are not allowed
 
 </div>
 
+{% configuration %}
+alias:
+  description: Friendly name for the script.
+  required: false
+  type: string
+description:
+  description: A description of the script that will be displayed in the Services tab under Developer Tools.
+  required: false
+  default: ''
+  type: string
+fields:
+  description: Information about the parameters that the script uses; see the [Passing variables to scripts](#passing-variables-to-scripts) section below.
+  required: false
+  default: {}
+  type: map
+  keys:
+    PARAMETER_NAME:
+      description: A parameter used by this script.
+      type: map
+      keys:
+        description:
+          description: A description of PARAMETER_NAME.
+          type: string
+        example:
```

Comment:
I have my doubts whether `example` is the correct name for the function it has, the name `value` I think is more appropriate 🤔

Reply:
That would need to be changed in https://github.com/home-assistant/home-assistant/pull/26056.

I was following the [services.yaml](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/homeassistant/services.yaml) format.